Plutarch/Api/V1/AssocMap.hs

```diff
@@ -105,12 +105,12 @@ plookup :: (PIsData k, PIsData v) => Term s (k :--> PMap _ k v :--> PMaybe v)
 plookup = phoistAcyclic $
   plam $ \key ->
     plookupDataWith
-      # (phoistAcyclic $ plam $ \pair -> pcon $ PJust $ pfromData $ psndBuiltin # pair)
+      # phoistAcyclic (plam $ \pair -> pcon $ PJust $ pfromData $ psndBuiltin # pair)
```

Comment:
?

Reply:
I want apply-redact for the whole repo :D 
Hlint is so mad at us.

Reply:
Please do this in another PR.